### PR TITLE
TXAD-154: Fix Upload Photo issue of Report New QA Screen.

### PR DESCRIPTION
**Pull request by ruchawajpe**

### CODE REVIEW

File: lib/blocs/pickAndUploadImage/pick_and_upload_image_bloc.dart [hunk 1]
```diff
@@ -37,6 +37,7 @@ class PickAndUploadImageBloc
 
   _uploadInitial(UploadInitial event, Emitter<PickAndUploadImageStates> emit) {
     isInitial = true;
+    number = 0;
```
Comment (objectorientguy) [inline lib/blocs/pickAndUploadImage/pick_and_upload_image_bloc.dart:40]: What is number? Page Number? Mobile Number? House Number? Plot Number? 🤔

File: lib/blocs/pickAndUploadImage/pick_and_upload_image_bloc.dart [hunk 2]
```diff
@@ -37,6 +37,7 @@ class PickAndUploadImageBloc
 
   _uploadInitial(UploadInitial event, Emitter<PickAndUploadImageStates> emit) {
     isInitial = true;
```
Comment (objectorientguy) [inline lib/blocs/pickAndUploadImage/pick_and_upload_image_bloc.dart:39]: What is isInitial? What is this variable talking about?

File: lib/blocs/pickAndUploadImage/pick_and_upload_image_bloc.dart [hunk 3]
```diff
@@ -60,6 +61,7 @@ class PickAndUploadImageBloc
         bool isAttached = false;
```
Comment (objectorientguy) [inline lib/blocs/pickAndUploadImage/pick_and_upload_image_bloc.dart:61]: isAttached - emotions, money, feelings, image, pdf? What is attached? 🤔

**Review by objectorientguy — CHANGES_REQUESTED** [review]

File: lib/blocs/pickAndUploadImage/pick_and_upload_image_bloc.dart (hunk 1 above)
Comment (ruchawajpe) [reply lib/blocs/pickAndUploadImage/pick_and_upload_image_bloc.dart:40]: It's the increment number of the image. For eg: if 1 image is uploaded, number will be 1, if 2 images are uploaded the number will be 2 etc.

File: lib/blocs/pickAndUploadImage/pick_and_upload_image_bloc.dart (hunk 2 above)
Comment (ruchawajpe) [reply lib/blocs/pickAndUploadImage/pick_and_upload_image_bloc.dart:39]: It tells if we are adding image for the first time.Eg: Initial State.

File: lib/blocs/pickAndUploadImage/pick_and_upload_image_bloc.dart [hunk 4]
```diff
@@ -21,8 +21,8 @@ class PickAndUploadImageBloc
       getIt<UploadImageRepository>();
   final ImagePicker _imagePicker = ImagePicker();
   final CustomerCache _customerCache = getIt<CustomerCache>();
-  bool isInitial = true;
-  int number = 0;
+  bool isUploadInitial = true;
```
Comment (objectorientguy) [inline lib/blocs/pickAndUploadImage/pick_and_upload_image_bloc.dart:24]: isUploadInitial -> isInitialUpload 
imageNumber -> imageIndex

**Review by objectorientguy — CHANGES_REQUESTED** [review]

File: lib/blocs/pickAndUploadImage/pick_and_upload_image_bloc.dart (hunk 4 above)
Comment (ruchawajpe) [reply lib/blocs/pickAndUploadImage/pick_and_upload_image_bloc.dart]: Done

**Review by objectorientguy — APPROVED** [review]

**Pull request merged**